### DISCUSSION TITLE
[Feature Request] Multiple accounts

### DISCUSSION THREAD

**Discussion by adishalilovic-abh** [community/community#39511] [Mobile] [answered]
Hi. Is there any idea of implementing ability to have multiple regular accounts?

**Comment by jfullstackdev** [comment]
actually, it's the reverse

GitHub encourages only one account

https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/merging-multiple-personal-accounts

what is possible is to add the emails that you need for your commits or keep those badges and commits in the past

**Reply by adishalilovic-abh** [reply]
That makes sense.
But, I am in situation where I use my personal account and have another one for work which I cannot merge.

**Reply by liebeslied** [reply]
Me too.
And if I'm not *really* careful about clearing cookies, the web UI gets confused about which one I'm signed into
(specifically I have this problem clicking on invitation-to-repository links; the wrong account ends up accepting the invitation even though I thought I was signed out of it, or it shows me the profile picture of the wrong account!)
I wish it could work like google, where I could be signed into both at once and not have to sign out and in and make sure I absolutely cleared everything to switch accounts, just to see a different repository I'm working on at the moment.

**Comment by QQuestio** [comment]
Hi

**Reply by QQuestio** [reply]
Ну що тут роби ь я не хакер

**Comment by AJ0545282** [comment]
I second this.

I have a personal account and a work account...

And no choice there, as it's company policy.

**Comment by iskae** [comment, marked as answer]
Hi @adishalilovic-abh 👋 

Thanks so much for reaching out! We have an existing internal tracking issue for this and I will be adding your request in there too 👍 

There is no timeline yet for supporting this, but please stay tuned for the future versions 🎉

**Comment by eliperkins** [comment]
Hey @adishalilovic-abh and folks, we've shipped functionality this week to allow you to log in with multiple accounts, like your personal and work accounts, at a the same time! [Read more about it in our Changelog post](https://github.blog/changelog/2023-08-15-log-in-with-multiple-github-accounts-on-github-mobile/).

<img width="1200" alt="image" src="https://github.com/community/community/assets/1051453/d2ba902e-2199-4625-948a-532d64a6f44e">

Download or update GitHub Mobile today from the [Apple App Store](https://apps.apple.com/us/app/github/id1477376905) or [Google Play Store](https://play.google.com/store/apps/details?id=com.github.android) to get started.

**Reply by adishalilovic-abh** [reply]
Hi @eliperkins. I can't express my facial expression when I read the release notes prior to the update. Thank you, @eliperkins and the Github Mobile team, for bringing it to life. It works incredibly well!
